### PR TITLE
Accumulator fixes

### PR DESCRIPTION
**Pull request by sumuditha-viraj**

### CODE REVIEW

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/langserver/util/resolvers/AnnotationAttachmentResolver.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
+package org.ballerinalang.composer.service.workspace.langserver.util.resolvers;
```
Comment (maheeka) [inline modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/langserver/util/resolvers/AnnotationAttachmentResolver.java]: License header missing

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/langserver/util/resolvers/PackageActionsAndFunctionsResolver.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package org.ballerinalang.composer.service.workspace.langserver.util.resolvers;
```
Comment (maheeka) [inline modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/langserver/util/resolvers/PackageActionsAndFunctionsResolver.java]: License header

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/langserver/util/resolvers/AnnotationAttachmentResolver.java (hunk 1 above)
Comment (sumuditha-viraj) [inline modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/langserver/util/resolvers/AnnotationAttachmentResolver.java]: Fixed in #2300

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/langserver/util/resolvers/PackageActionsAndFunctionsResolver.java (hunk 2 above)
Comment (sumuditha-viraj) [inline modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/langserver/util/resolvers/PackageActionsAndFunctionsResolver.java]: Fixed in #2300

**Pull request merged**